rasa/nlu/classifiers/fallback_classifier.py

```diff
@@ -47,10 +54,55 @@ def process(self, message: Message, **kwargs: Any) -> None:
         message.data[INTENT_RANKING_KEY].insert(0, _fallback_intent())
 
     def _should_fallback(self, message: Message) -> bool:
-        return (
-            message.data[INTENT].get(INTENT_CONFIDENCE_KEY)
-            < self.component_config[THRESHOLD_KEY]
-        )
+        """Check if the fallback intent should be predicted.
+
+        Args:
+            message: The current message and its intent predictions.
+
+        Returns:
+            `True` if the fallback intent should be predicted.
+        """
+        intent_name = message.data[INTENT].get("name")
```

Comment:
Minor: As we have a constant for "confidence" does it make sense to also introduce one for "name"?

Reply:
done and tried to replace it in most files